### PR TITLE
added docs

### PR DESCRIPTION
**Pull request by suryavanshi08**
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1424)

### CODE REVIEW

**Review by jarifibrahim — APPROVED** [review]
I haven't tested the changes locally but I believe @suryavanshi08 has moved the docs from dgraph to badger without any changes.

File: docs/themes/hugo-docs/layouts/_default/article.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{ $currentVersion := getenv "CURRENT_VERSION" }}
+{{ $versionString := getenv "VERSIONS" }}
+{{ $versions := split $versionString "," }}
+{{ $latestVersion := index $versions 0 }}
+
+{{ if (eq $currentVersion "master") }}
+  <div class="alert alert-warning">
+    <i class="fa fa-warning"></i> You are looking at the docs for the unreleased <code>master</code> branch of Dgraph. The latest version is <a href="{{ .Site.BaseURL }}/..">{{ $latestVersion }}</a>.
```
Comment (jarifibrahim) [inline docs/themes/hugo-docs/layouts/_default/article.html:8]: This looks out of place. Why is there a mention of dgraph docs on the badger docs page?

File: docs/themes/hugo-docs/layouts/_default/article.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  </div>
+{{ else if not (eq $latestVersion $currentVersion) }}
+  <div class="alert alert-warning">
+    <i class="fa fa-warning"></i> You are looking at the docs for an older version of Dgraph ({{ $currentVersion }}). The latest version is <a href="{{ .Site.BaseURL }}/..">{{ $latestVersion }}</a>.
```
Comment (jarifibrahim) [inline docs/themes/hugo-docs/layouts/_default/article.html:12]: This looks out of place. Why is there a mention of dgraph docs on the badger docs page?

File: docs/themes/hugo-docs/layouts/partials/header.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <script async src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.25.0/addon/edit/closebrackets.min.js" integrity="sha256-qhu2a7DSpQaammMaLw1iruxBpxmgdw8mdQaQWQUokcY=" crossorigin="anonymous"></script>
+    <script async src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.25.0/mode/javascript/javascript.min.js" integrity="sha256-7Bdg/UdDMmHDPafpQDYZNPCEbnRP7eAvu+2hEoqRCXs=" crossorigin="anonymous"></script>
+    <!-- END: Required by runnable -->
+
+    <script>
+      {{ $currentBranch := getenv "CURRENT_BRANCH" }}
+      {{ $dgraphEndpoint := getenv "DGRAPH_ENDPOINT" }}
+
+      {{/* Set dgraph endpoint depending on the DGRAPH_ENDPOINT environment variable in the script local.sh to ensure that runnables work fine */}}
+      window.DGRAPH_ENDPOINT = {{ if (eq $dgraphEndpoint "" ) }}"https://play.dgraph.io/query?latency=true"{{ else }}{{ $dgraphEndpoint }}{{ end }}
```
Comment (jarifibrahim) [inline docs/themes/hugo-docs/layouts/partials/header.html:46]: Why is this here?

File: docs/themes/hugo-docs/layouts/partials/header.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    <!-- START: Required by runnable -->
+    <script src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.25.0/codemirror.js" integrity="sha256-x3BtPVERvw1ozPlwivuFLZoViFCeAAs1Gt6SxEedmo8=" crossorigin="anonymous"></script>
+    <link href="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.25.0/codemirror.css" rel="stylesheet" integrity="sha256-7sZe9dsqyNZC9Mk+WPRB7nVvPPPXyorzm2QKqlgXC4k=" crossorigin="anonymous" />
+    <script async src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.25.0/addon/edit/closebrackets.min.js" integrity="sha256-qhu2a7DSpQaammMaLw1iruxBpxmgdw8mdQaQWQUokcY=" crossorigin="anonymous"></script>
+    <script async src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.25.0/mode/javascript/javascript.min.js" integrity="sha256-7Bdg/UdDMmHDPafpQDYZNPCEbnRP7eAvu+2hEoqRCXs=" crossorigin="anonymous"></script>
+    <!-- END: Required by runnable -->
+
+    <script>
+      {{ $currentBranch := getenv "CURRENT_BRANCH" }}
+      {{ $dgraphEndpoint := getenv "DGRAPH_ENDPOINT" }}
```
Comment (jarifibrahim) [inline docs/themes/hugo-docs/layouts/partials/header.html:43]: Do we even need this?

File: docs/themes/hugo-docs/layouts/shortcodes/runnable.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+              <pre><code class="no-copy"><span class="query-content">{{ $code }}</span></code></pre>
+
+            </div>
+
+            <div class="runnable-tab-content" data-tab="edit">
+              <textarea class="query-content-editable">{{ $code }}</textarea>
+            </div>
+
+            <div class="runnable-tab-content" data-tab="curl">
+                {{ partial "example-curl" ( dict "Content" . "Code" "blahblah" "Vars" ( .Get "vars" ) ) }}
```
Comment (jarifibrahim) [inline docs/themes/hugo-docs/layouts/shortcodes/runnable.html:55]: Why do we have example curl in badger documentation?

File: docs/themes/hugo-docs/layouts/shortcodes/runnable.html [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+          <div class="row footer">
+            <div class="col-12">
+              <div class="actions pull-right">
+                <div class="edit-actions">
+                  <a class="code-btn" data-action="reset" href="#">Reset</a>
+                  <a class="code-btn" data-action="discard" href="#">Discard</a>
+                  <a class="code-btn" data-action="save" href="#"><i class="fa fa-save"></i> Save</a>
+                </div>
+                <div class="normal-actions">
+                  <a class="code-btn" data-action="edit" href="#"><i class="fa fa-edit"></i> Edit query</a>
```
Comment (jarifibrahim) [inline docs/themes/hugo-docs/layouts/shortcodes/runnable.html:83]: Why is there a `edit query` button? Actually, why do we even have the runnable file? Badger does not have any runnable code.

File: docs/themes/hugo-docs/static/css/runnable.css [hunk 7]
```diff
@@ -0,0 +1,290 @@
+/**
+ * CSS for runnable component
```
Comment (jarifibrahim) [inline docs/themes/hugo-docs/static/css/runnable.css:2]: Do we need this runnable css file?

**Review by jarifibrahim — COMMENTED** [review]
Added some comments.

File: docs/themes/hugo-docs/layouts/partials/header.html (hunk 3 above)
Comment (suryavanshi08) [reply docs/themes/hugo-docs/layouts/partials/header.html]: I have removed this code.

File: docs/themes/hugo-docs/layouts/partials/header.html (hunk 4 above)
Comment (suryavanshi08) [reply docs/themes/hugo-docs/layouts/partials/header.html]: I have removed this code.

File: docs/themes/hugo-docs/layouts/shortcodes/runnable.html (hunk 6 above)
Comment (suryavanshi08) [reply docs/themes/hugo-docs/layouts/shortcodes/runnable.html]: I have deleted this file along with runnable.css and runnable.js file as we don't require this if we don’t have any runnable code.

File: docs/themes/hugo-docs/static/css/runnable.css (hunk 7 above)
Comment (suryavanshi08) [reply docs/themes/hugo-docs/static/css/runnable.css]: I have removed this file.
